articles/hardware/hs64/overview.md

```diff
@@ -0,0 +1,29 @@
+---
+uid: hs64_hs64
+title: Headstage 64
+---
+
+These are the devices available on the Headstage64:
+
+- [Rhd2164](xref:hs64_rhd2164):
+    - 64 electrophysiology channels for passive probes (e.g. tetrode, silicon probe, etc.) sampled at 30 kHz with 16 bit depth
+    - Adjustable analog band-pass filter:
+      - Lower cutoff configurable from 0.1 Hz to 500 Hz
+      - Upper cutoff configurable from 100 Hz to 20 kHz
+    - Optional adjustable digital high-pass filter with cutoff configurable from 0.146 Hz to 3309 Hz
+    - Three auxiliary ADC channels sampled at 30 kHz with 16 bit depth
+- [Bno055](xref:hs64_bno055): 9-axis IMU for real-time, 3D orientation tracking, and easy automated commutation with Open Ephys commutators
+- [Ts4231](xref:hs64_ts4231): For compatibility with HTC Vive Lighthouses for real-time, 3D position tracking
```

Comment:
add sampling rate for Bno055 and TS4231 ?

Reply:
I don't know the ts4231 sample rate. I made an issue #87 to fix in this future once I know the sample rate so that I can resolve this conversation and we can merge this to main in the meanwhile.